glib/src/signal.rs

```diff
@@ -13,6 +13,36 @@ use std::mem;
 use std::num::NonZeroU64;
 
 /// The id of a signal that is returned by `connect`.
+///
+/// ```rust
+/// use glib::SignalHandlerId;
+/// use gtk::prelude::*;
```

Comment:
You're in glib, so this won't work when running `cargo test`. Maybe try with a signal from glib?

Reply:
There are no signals in GLib :)

Reply:
So I think this has to be `no_compile` unfortunately